### ISSUE TITLE
Could not parse metadata

### ISSUE THREAD

**Issue by trungng2006** [hadynz/obsidian-kindle-plugin#15]
The plugin stuck on `starting sync`, and it show this error in the console:
```
Uncaught (in promise) Error: Could not parse metadata of: Thinking, Fast and Slow (Kahneman, Daniel)
    at t.KindleEntryParsed.parseMetadata (eval at <anonymous> (app.js:1), <anonymous>:2:1677)
    at new t.KindleEntryParsed (eval at <anonymous> (app.js:1), <anonymous>:2:1186)
    at eval (eval at <anonymous> (app.js:1), <anonymous>:2:5056)
    at Array.forEach (<anonymous>)
    at Object.t.parseKindleEntries (eval at <anonymous> (app.js:1), <anonymous>:2:5036)
    at eval (eval at <anonymous> (app.js:1), <anonymous>:2:476397)
    at ct.startSync (eval at <anonymous> (app.js:1), <anonymous>:2:476470)
```

I am using the upload Myclipping.txt method

**Comment by hadynz** [comment]
One of the challenges when attempting to parse the `My Clippings.txt` file is there is no definition to how data can be stored in this file. It seems that your specific file contains data that the plugin has not catered for.

Some immediate improvement that I will make as a resul:

1. Skip any book that could no be parsed, and show you an error. Right now the plugin tries to parse all records in your file in an all or nothing manner
2. Add better logging to console of the bad data that could not be parsed to make troubleshooting easier

Once I implement step 2, you should be able to see a more detailed error message that will guide me to what fix I need to make.

Alternatively, if you are open to it (and this is totally if you are comfortable with this) is share with me your `My Clippings.txt` file. I will immediately be able to troubleshoot, fix and test the problem and release a new version that will work for your data. If you are open to this idea, you can share your file on Google Drive/Dropbox, or email it to me (hadyos@gmail.com).

**Comment by hadynz** [comment]
Thanks for sharing your Clippings file. I found the issue and resolved it in [0.1.1][1].

[1]: https://github.com/hadynz/obsidian-kindle-plugin/releases/tag/0.1.1
